### ISSUE TITLE
Musixmatch synced lyrics integration

### ISSUE THREAD

**Issue by shahar015** [th-ch/youtube-music#650]
It would be awsome if in the future there'll be an option to see floating synced lyrics from musixmatch

**Comment by SaqinNoor** [comment]
Stumbled upon this GreaseMonkey script to achieve this: https://github.com/ProComGameS/ytm-lyrics-v2/tree/ProComGameS-YTMLyrics-V2

Any way to implement this in?

Someone is also developing a chrome extension similar to the above (https://www.reddit.com/r/YoutubeMusic/comments/14z006v/synced_lyrics_on_yt_music_web/)

**Comment by ArjixWasTaken** [comment]
> So, will we ever have this feature?

😕
https://github.com/th-ch/youtube-music/pull/2207

**Comment by mnofrizal** [comment]
any news about floating lyrics?

**Comment by ArjixWasTaken** [comment]
> any news about floating lyrics?

Define floating lyrics.

Are you talking about lyrics synced to the word like karaoke?

**Comment by mnofrizal** [comment]
> > any news about floating lyrics?
> 
> Define floating lyrics.
> 
> Are you talking about lyrics synced to the word like karaoke?
> 
> Or are you talking about literally floating lyrics on a popup window?

Surely there are many in here who works by listening to songs from YouTube, how about implementing floating lyrics (on top or pip) so we can works and sing together

**Comment by mnofrizal** [comment]
> So you are talking about a popup window.

yeah maybe like that

**Comment by ArjixWasTaken** [comment]
> > So you are talking about a popup window.
> 
> yeah maybe like that

Would you be satisfied with the current synced lyrics but on their own separate window?

**Comment by mnofrizal** [comment]
> Would you be satisfied with the current synced lyrics but on their own separate window?

yes! if we have separetd window it will be great, mostly i minimize the app so i cant see the lyric when coding

**Comment by JellyBrick** [comment]
See https://github.com/organization/alspotron/blob/master/README-i18n/en.md

**Comment by JellyBrick** [comment]
@itzmanish see https://github.com/organization/alspotron/blob/master/README-i18n/en.md

**Comment by iambryancs** [comment]
A floating lyrics similar to [MiniLyrics ](https://www.crintsoft.com/) would be great.
I think it's possible to reuse the current lyrics related plugins but make it floating and transparent.

MiniLyrics with WMplayer - transparent, scrolling lyrics.
![Image](https://github.com/user-attachments/assets/efbe4737-78ba-4688-b925-f083cc427bee)

MiniLyrics with WMplayer - hovered. We probably don't need this if the plugin's controls and customizations are within the main app's interface.
![Image](https://github.com/user-attachments/assets/bd4020f1-ee2c-4ed9-a3e3-4862e8435ed7)

**Comment by mnofrizal** [comment]
> A floating lyrics similar to [MiniLyrics ](https://www.crintsoft.com/) would be great. I think it's possible to reuse the current lyrics related plugins but make it floating and transparent.
> 
> MiniLyrics with WMplayer - transparent, scrolling lyrics. ![Image](https://github.com/user-attachments/assets/efbe4737-78ba-4688-b925-f083cc427bee)
> 
> MiniLyrics with WMplayer - hovered. We probably don't need this if the plugin's controls and customizations are within the main app's interface. ![Image](https://github.com/user-attachments/assets/bd4020f1-ee2c-4ed9-a3e3-4862e8435ed7)

That's what we want, it will ne great if amy extestion or features like that
